nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
+license agreements. See the NOTICE file distributed with this work for additional
+information regarding copyright ownership. The ASF licenses this file to
+You under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+by applicable law or agreed to in writing, software distributed under the
+License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+OF ANY KIND, either express or implied. See the License for the specific
+language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>nifi-elasticsearch-bundle</artifactId>
+        <groupId>org.apache.nifi</groupId>
+        <version>0.4.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>nifi-elasticsearch-processors</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <slf4jversion>1.7.12</slf4jversion>
+        <es.version>2.1.0</es.version>
```

Comment:
@mattyb149 This is a minor thing, but with the rate ES has been progressing I'd worry about whether enough people are using 2.x in production. I know upgrades/incompatibility from 1.6 -> 1.7 -> 2.x haven't been too terrible, but I'd be surprised if big installs have made the leap already.